include/gz/gui/qml/GzCard.qml

```diff
@@ -94,32 +94,31 @@ Pane {
    */
   property string cardBackground: "#00000000"
 
-  /**
-   *
-   */
-  property var backgroundItem: null
-
-  /**
-   * Stores last height of plugin to expand to.
-   */
-  property int lastHeight: 50
-
   /**
    * True if there's at least one anchor set for the card.
    * There's no way to check the anchors themselves, so we need
    * to keep track of this ourselves.
    */
   property bool anchored: false
 
+  /**
+   * Keep track of whether the card is floating
+   */
+  property bool floating: false
+
   /**
    * Minimum width of the card pane
    */
-  property int cardMinimumWidth: 250;
+  property alias cardMinimumWidth: content.minimumWidth;
```

Comment:
I think this we need to update the test for this change:

```
  /github/workspace/src/Plugin_TEST.cc:304: Failure
  Expected equality of these values:
    propElem->Attribute("type")
      Which is: "double"
    pluginTypes[propElem->Attribute("key")]
      Which is: "int"
```

Reply:
Fixed in https://github.com/gazebosim/gz-gui/pull/651/commits/2570a75ba26a51926f7c07cd0adf6804ee2b899e

I opted to ignore `card*` properties from being parsed from XML since that wasn't working anyway. Looking at the implementation, I don't think they were intended to be exposed since they are updated internally to collapse/expand the card.